src/ui/setup/dynamic/devtools.ts

```diff
@@ -104,7 +116,12 @@ const SessionErrorMessages: Record<number, Partial<UnexpectedError>> = {
     content: "Our servers hiccuped but things should be back to normal soon.",
   },
   [SessionError.UnknownFatalError]: {
-    content: "Refreshing should help.\nIf not, please try recording again.",
+    content:
+      isWindows && isFirefox
+        ? "Our Windows browser is currently being upgraded to Chrome, which will reduce errors like these. Thanks for your patience!"
```

Comment:
> Our Windows browser is currently being upgraded to Chrome 
This leads with our "fix" as opposed to explaining the problem.

We should succinctly describe the problem and then share a solution. With Windows, that's wait till Chrome. With other runtimes, thats reach out on Discord.

Here's a fast version

> The browser replayed an event out of order. We are hoping to release a new Chrome based browser in a couple of months  which will be more reliable. Thanks for your patience 🙏